build/publish/Publish.csproj

```diff
@@ -0,0 +1,186 @@
+<Project>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <!-- TFM doesn't matter. These settings are required to make NuGet happy so we can restore required MSBuild packages. -->
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <ManifestsPath>$(ArtifactsDir)manifests\</ManifestsPath>
+    <MaestroApiEndpoint Condition="'$(MaestroApiEndpoint)' == ''">https://maestro-prod.westus2.cloudapp.azure.com</MaestroApiEndpoint>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Feed" Version="2.2.0-beta.19061.6" />
+    <PackageReference Include="Microsoft.DotNet.Maestro.Tasks" Version="1.1.0-beta.19064.1" />
+  </ItemGroup>
+
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <PublishDependsOn>
+      GetFilesToPublish;
+      GenerateBuildAssetManifest;
+      PublishToAzureFeed;
+      PublishToMyGet;
+      PublishToBuildAssetRegistry;
+    </PublishDependsOn>
+  </PropertyGroup>
+
+  <Target Name="Publish" DependsOnTargets="$(PublishDependsOn)" />
+
+  <Target Name="GetFilesToPublish">
+    <ItemGroup>
+      <!-- Installer output files with specific metadata. -->
+      <_FilesToPublish Include="$(InstallersOutputPath)*.txt">
+        <ContentType>text/plain</ContentType>
+      </_FilesToPublish>
+
+      <_FilesToPublish Include="$(InstallersOutputPath)*.version">
+        <ContentType>text/plain</ContentType>
+        <CacheControl>no-cache, no-store, must-revalidate</CacheControl>
+      </_FilesToPublish>
+
+      <_FilesToPublish Include="$(InstallersOutputPath)*.svg">
+        <CacheControl>no-cache, no-store, must-revalidate</CacheControl>
+        <ContentType>image/svg+xml</ContentType>
+      </_FilesToPublish>
+
+      <!-- All other installer files. -->
+      <_FilesToPublish Include="$(InstallersOutputPath)*" Exclude="@(_FilesToPublish)" />
+
+      <!-- Java packages -->
+      <_FilesToPublish Include="$(ProductPackageOutputPath)*.jar;$(ProductPackageOutputPath)*.pom">
+        <BlobBasePath>aspnetcore/jar/$(PackageVersion)/</BlobBasePath>
+      </_FilesToPublish>
+
+      <!-- NPM packages -->
+      <NpmPackageToPublish Include="$(ProductPackageOutputPath)*.tgz">
+        <BlobBasePath>aspnetcore/npm/$(PackageVersion)/</BlobBasePath>
+      </NpmPackageToPublish>
+      <_FilesToPublish Include="@(NpmPackageToPublish)" />
```

Comment:
The `npm` files weren't previously published to the Azure feed. Is this change intentional?

Reply:
Yes. As we prepare to move to Azure Pipelines, we need to make sure everything ends up in the blob store since we won't have network drop shares